### DISCUSSION TITLE
Donating to this project via ads without credit or debit card

### DISCUSSION THREAD

**Discussion by Uniquenameihave** [FossifyOrg/General-Discussion#60] [General]
How can I contribute financially to fossify but I don't have financial freedom(I'm a kid)  which means that I don't have my own credit or debit card to donate to this project 
Is there any way that by me  watching ads on any website or something similar so  the developers can get money for this project ?

**Comment by Aga-C** [comment]
It's not the answer to your question, but can you edit the title of the discussion? Nobody will know what the discussion is about if there is just a dot.

EDIT: Thanks!

**Comment by inson1** [comment]
hey maybe would be better if you learn programming, you could help with development and learn good skill
watching ads doesnt pay very well (per one person)

**Reply by Uniquenameihave** [reply]
Which programming language will your recommend me to learn for this project or any other similar projects ? 
Should I start with python?

**Reply by inson1** [reply]
@Uniquenameihave Android apps are mostly build with Kotlin (and all Fossify/SMT apps too). 
I dont think it matters too much, where you start(but for start stay with high level languages). It depends what do you want to build and where are more resources.
Python has lot for resources for beginners, but I didnt yet seen any android app built with it. It doesnt matter if you build some easy apps with Python and then move to Kotlin or start with Kotlin. Just pick some really easy idea and build that. But ofc first app should be Hello world from tutorial and second app should be some easy app with terminal or gui from another tutorial.
Just **be aware of the tutorial hell**. And dont be stuck on choosing the best programming language, it doesnt matter, **just build something**.
But yes tutorials are important.

**Comment by y1111** [comment]
the project could put crypto adresses for receiving donation . specially  those with low transaction fees like doge, xrp, ada, etc .

**Reply by inson1** [reply]
done and done, in org readme

**Comment by naveensingh** [comment]
Closing it now as the original question has been answered.
